modules/tensor_mechanics/doc/content/documentation/systems/Postprocessors/MaterialTimeStepPostprocessor.md

```diff
@@ -59,6 +61,39 @@ Initially the value of the maximum time step size is set to `std::numeric_limits
 Once the inelastic material model begins to calculate inelastic strain, the value
 of the allowable time step size varies with the inelastic strain computation.
 
+The limiting time step size is stored within a material property called `matl_timestep_limit`.
+This material property can be set up either by the 
+[ComputeMultipleInelasticStress](/Materials/ComputeMultipleInelasticStress.md) model (for creep
+or plastic strains) or the [ComputeDamageStress](/Materials/ComputeDamageStress.md) for 
+continuum damage models.
+
+### Minimum Time Step Size Based on Number of Elements Changed
+
+The allowable time step size can also be controlled by the total number of elements changed
```

Comment:
total number of elements changed -> total number of elements for which a material property changes